root/tree/dataframe/df104_HiggsToTwoPhotons.py

```diff
@@ -0,0 +1,212 @@
+    data.SetMarkerSize(1.2)
+    data.SetLineWidth(2)
+    data.SetLineColor(ROOT.kBlack)
+    data.Draw("E SAME")
+    data.SetMinimum(1e-3)
+    data.SetMaximum(8e3)
+    data.GetYaxis().SetLabelSize(0.045)
+    data.GetYaxis().SetTitleSize(0.05)
+    data.SetStats(0)
+    data.SetTitle("")
+
+    # Scale simulated events with luminosity * cross-section / sum of weights
+    # and merge to single Higgs signal
+    lumi = 10064.0
+    ggh.Scale(lumi * 0.102 / 55922617.6297)
+    vbf.Scale(lumi * 0.008518764 / 3441426.13711)
+    higgs = ggh.Clone()
+    higgs.Add(vbf)
+    higgs.Draw("HIST SAME")
+
+    # Draw ratio
+    lower_pad.cd()
+
+    ratiobkg = ROOT.TF1("zero", "0", 105, 160)
+    ratiobkg.SetLineColor(4)
+    ratiobkg.SetLineStyle(2)
+    ratiobkg.SetLineWidth(2)
+    ratiobkg.SetMinimum(-125)
+    ratiobkg.SetMaximum(250)
+    ratiobkg.GetXaxis().SetLabelSize(0.08)
+    ratiobkg.GetXaxis().SetTitleSize(0.12)
+    ratiobkg.GetXaxis().SetTitleOffset(1.0)
+    ratiobkg.GetYaxis().SetLabelSize(0.08)
+    ratiobkg.GetYaxis().SetTitleSize(0.09)
+    ratiobkg.GetYaxis().SetTitle("Data - Bkg.")
+    ratiobkg.GetYaxis().CenterTitle()
+    ratiobkg.GetYaxis().SetTitleOffset(0.7)
+    ratiobkg.GetYaxis().SetNdivisions(503, False)
+    ratiobkg.GetYaxis().ChangeLabel(-1, -1, 0)
+    ratiobkg.GetXaxis().SetTitle("m_{#gamma#gamma} [GeV]")
+    ratiobkg.Draw()
+
+    ratiosig = ROOT.TH1F("ratiosig", "ratiosig", 5500, 105, 160)
+    ratiosig.Eval(fit)
+    ratiosig.SetLineColor(2)
+    ratiosig.SetLineStyle(1)
+    ratiosig.SetLineWidth(2)
+    ratiosig.Add(bkg, -1)
+    ratiosig.Draw("SAME")
+
+    ratiodata = data.Clone()
+    ratiodata.Add(bkg, -1)
+    ratiodata.Draw("E SAME")
+    for i in range(1, data.GetNbinsX()):
+        ratiodata.SetBinError(i, data.GetBinError(i))
+
+    # Add legend
+    upper_pad.cd()
+    legend = ROOT.TLegend(0.55, 0.55, 0.89, 0.85)
+    legend.SetTextFont(42)
+    legend.SetFillStyle(0)
+    legend.SetBorderSize(0)
+    legend.SetTextSize(0.05)
+    legend.SetTextAlign(32)
+    legend.AddEntry(data, "Data" ,"lep")
+    legend.AddEntry(bkg, "Background", "l")
+    legend.AddEntry(fit, "Signal + Bkg.", "l")
+    legend.AddEntry(higgs, "Signal", "l")
+    legend.Draw("SAME")
+
+    # Add ATLAS label
+    text = ROOT.TLatex()
+    text.SetNDC()
+    text.SetTextFont(72)
+    text.SetTextSize(0.05)
+    text.DrawLatex(0.18, 0.84, "ATLAS")
+    text.SetTextFont(42)
+    text.DrawLatex(0.18 + 0.13, 0.84, "Open Data")
+    text.SetTextSize(0.04)
+    text.DrawLatex(0.18, 0.78, "#sqrt{s} = 13 TeV, 10 fb^{-1}");
+
+    # Save the plot
+    c.SaveAs("HiggsToTwoPhotons.pdf");
+
+
+path = os.environ["RB_DATASETDIR"]
+
+
+def test_df104_HiggsToTwoPhotons_noimt(benchmark):
+    benchmark.pedantic(payload, kwargs={'path': path, 'enable_imt': True}, iterations=1, rounds=1)
+
+
+def test_df104_HiggsToTwoPhotons_imt(benchmark):
+    benchmark.pedantic(payload, kwargs={'path': path, 'enable_imt': False}, iterations=1, rounds=1)
```

Comment:
the 'enable_imt' arguments are switched w.r.t. the benchmark names

Reply:
aaaah thanks!

Reply:
Fixed.